README.md

```diff
@@ -111,6 +112,26 @@ You can run `rustfmt --help` for more information.
 * [Atom](atom.md)
 * Visual Studio Code using [RustyCode](https://github.com/saviorisdead/RustyCode) or [vsc-rustfmt](https://github.com/Connorcpu/vsc-rustfmt)
 
+## Checking style on a CI server
+
+To keep your code base consistently formatted, it can be helpful to fail the CI build
+when a pull request contains unformatted code. Using `--write-mode=diff` you can instruct
```

Comment:
the english here is a little confusing. Maybe something like " Using `--write-mode=digg` instructs Rustfmt to exit with an error code if the input is not formatted correctly. It will ..."